src/filterShows.ts

```diff
@@ -0,0 +1,15 @@
+import { Show } from "./Show";
+import shows from "./shows.json";
+
+export function filterShows(message: string): Show[] {
+  const filteredShows = shows.filter((s) =>
+    s.summary === null
+      ? s.name.toUpperCase().includes(message.toUpperCase()) ||
+        s.genres.join().toUpperCase().includes(message.toUpperCase())
```

Comment:
is this part necessary? as you have OR statements if summary is null it will just look at .name and .genres anyway